src/main/resources/templates/acs-status.ftl

```diff
@@ -1,36 +1 @@
-98<#rt>
-${ACSStatus.onLineStatus}<#rt>
-${ACSStatus.checkinOk}<#rt>
-${ACSStatus.checkoutOk}<#rt>
-${ACSStatus.acsRenewalPolicy}<#rt>
-${ACSStatus.statusUpdateOk}<#rt>
-${ACSStatus.offLineOk}<#rt>
-${ACSStatus.timeoutPeriod}<#rt>
-${ACSStatus.retriesAllowed}<#rt>
-${formatDateTime(ACSStatus.dateTimeSync, "yyyyMMdd    HHmmss")}<#t>
-${ACSStatus.protocolVersion}|<#rt>
-AO${ACSStatus.institutionId}|<#rt>
-AM${ACSStatus.libraryName}|<#rt>
-BX<@supportedMessages />|<#rt>
-AN${ACSStatus.terminalLocation}|<#rt>
-AF${ACSStatus.screenMessage}|<#rt>
-AG${ACSStatus.printLine}|<#rt>
-
-<#macro supportedMessages>
- ${PackagedSupportedMessages.patronStatusRequest}<#t>
- ${PackagedSupportedMessages.checkOut}<#t>
- ${PackagedSupportedMessages.checkIn}<#t>
- ${PackagedSupportedMessages.blockPatron}<#t>
- ${PackagedSupportedMessages.scAcsStatus}<#t>
- ${PackagedSupportedMessages.requestScAcsResend}<#t>
- ${PackagedSupportedMessages.login}<#t>
- ${PackagedSupportedMessages.patronInformation}<#t>
- ${PackagedSupportedMessages.endPatronSession}<#t>
- ${PackagedSupportedMessages.feePaid}<#t>
- ${PackagedSupportedMessages.itemInformation}<#t>
- ${PackagedSupportedMessages.itemStatusUpdate}<#t>
- ${PackagedSupportedMessages.patronEnable}<#t>
- ${PackagedSupportedMessages.hold}<#t>
- ${PackagedSupportedMessages.renew}<#t>
- ${PackagedSupportedMessages.renewAll}<#t>
-</#macro>
+98<#rt>
${ACSStatus.onLineStatus}<#rt>
${ACSStatus.checkinOk}<#rt>
${ACSStatus.checkoutOk}<#rt>
${ACSStatus.acsRenewalPolicy}<#rt>
${ACSStatus.statusUpdateOk}<#rt>
${ACSStatus.offLineOk}<#rt>
${ACSStatus.timeoutPeriod}<#rt>
${ACSStatus.retriesAllowed}<#rt>
${formatDateTime(ACSStatus.dateTimeSync, "yyyyMMdd    HHmmss")}<#t>
${ACSStatus.protocolVersion}|<#rt>
AO${ACSStatus.institutionId}|<#rt>
AM${ACSStatus.libraryName}|<#rt>
BX<@supportedMessages />|<#rt>
AN${ACSStatus.terminalLocation}|<#rt>
AF${ACSStatus.screenMessage}|<#rt>
AG${ACSStatus.printLine}|<#rt>

<#macro supportedMessages>
 ${PackagedSupportedMessages.patronStatusRequest}<#t>
 ${PackagedSupportedMessages.checkOut}<#t>
 ${PackagedSupportedMessages.checkIn}<#t>
 ${PackagedSupportedMessages.blockPatron}<#t>
 ${PackagedSupportedMessages.scAcsStatus}<#t>
 ${PackagedSupportedMessages.requestScAcsResend}<#t>
 ${PackagedSupportedMessages.login}<#t>
 ${PackagedSupportedMessages.patronInformation}<#t>
 ${PackagedSupportedMessages.endPatronSession}<#t>
 ${PackagedSupportedMessages.feePaid}<#t>
 ${PackagedSupportedMessages.itemInformation}<#t>
 ${PackagedSupportedMessages.itemStatusUpdate}<#t>
 ${PackagedSupportedMessages.patronEnable}<#t>
 ${PackagedSupportedMessages.hold}<#t>
 ${PackagedSupportedMessages.renew}<#t>
 ${PackagedSupportedMessages.renewAll}<#t>
</#macro>

```

Comment:
GitHub doesn't like this format at all. Why do we need to use this format for this file? I think the template should be independent of the message delimiter, so we should be free to use whatever line delimiter we want, i.e. not this one. 😄

Reply:
I found options to set line separators in IntelliJ so I set it to \r and and apparently it screwed up the whole file. I can set it back to \n

Reply:
Restored file